src/views/Auth.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useState } from 'react';
+import { signInUser, signupUser } from '../services/users';
+
+export default function Auth(items) {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [type, setType] = useState('signin');
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    setErrorMessage('');
+    if (!email || !password) return;
+    try {
+      const user =
+        type === 'signin' ? await signInUser(email, password) : await signupUser(email, password);
+      items.setCurrentUser(user.email);
+    } catch (e) {
+      e.message ? setErrorMessage(e.message) : setErrorMessage('Something went wrong. Refresh!');
+    }
+  };
+
+  return (
+    <div className="auth">
+      <h3 className={type === 'signin' ? 'active' : ''} onClick={() => setType('signin')}>
+        Sign In
+      </h3>
+      <h3 className={type === 'signup' ? 'active' : ''} onClick={() => setType('signup')}>
+        Sign Up
+      </h3>
```

Comment:
Remember to add css styling so that the user knows if they are signing in or up.